scripts/ethereum/main.go

```diff
@@ -235,21 +233,21 @@ func lock() []byte {
 		fmt.Println("query account failed", err)
 		return nil
 	}
-	//acc := accReply.Accounts[0]
-	acc := keys.Address{}
-	err = acc.UnmarshalText([]byte("0x416e9cc0abc4ea98b4066823a62bfa6515180582"))
-	if err != nil {
-		return nil
-	}
-	wallet, err := accounts.NewWalletKeyStore("/home/tanmay/Codebase/Test/WalletStore/keystore")
-	if err != nil {
-		return nil
-	}
-	wallet.Open(acc, "123")
+	acc := accReply.Accounts[0]
```

Comment:
Looks like unrelated changes

Reply:
this is the test scripts for ethereum lock and redeem, I merged it from Tanmay's changes so I can test.